### PR TITLE
Update changelog and version for v0.21.0 release.

### PR DESCRIPTION
**Pull request by zach-klippenstein**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,65 @@
...
+
+ * Eliminate `ExperimentalWorkflowUi`. (#565)
+ * Rename `Worker<T>` to `Worker<OutputT>`. (#570)
+ * Fix "java" being printed instead of class name in Rx2 Reactors. (#607)
+ * Fix a worker crash introduced by `onReceiveOrClosed`. (#630)
+ * Make `onEvent`, `makeActionSink`, and `makeEventSink` include what action was lost to when they receive more than one event. (#673)
+
+### Swift
+
+ * Apply child workflow output as an action in `RenderTester`. (#595)
```
Comment (rjrjr) [inline CHANGELOG.md]: Didn't we also bump the `ReactiveSwift` version?

Comment (zach-klippenstein) [reply CHANGELOG.md]: Yes we did! Added.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,65 @@
...
+#### Other changes:
+
+ * Eliminate `ExperimentalWorkflowUi`. (#565)
+ * Rename `Worker<T>` to `Worker<OutputT>`. (#570)
+ * Fix "java" being printed instead of class name in Rx2 Reactors. (#607)
+ * Fix a worker crash introduced by `onReceiveOrClosed`. (#630)
+ * Make `onEvent`, `makeActionSink`, and `makeEventSink` include what action was lost to when they receive more than one event. (#673)
+
+### Swift
+
```
[the remark this answers is not in the document]
Comment (zach-klippenstein) [reply CHANGELOG.md]: Adopted your text with slightly different formatting.

**Comment by zach-klippenstein** [comment]
Merging eagerly, this has already passed a couple times and passes lint locally.

**Pull request merged**